lib/arke/strategy/base.rb

```diff
@@ -0,0 +1,71 @@
+require 'exchange'
+
+module Arke::Strategy
+  class Base
+    RunNotOverriden = Class.new(StandardError)
+
+    def initialize
+      @shutdown = false
+      # we limit strategy update rate here (seconds)
+      # easy to change if we will implement some stress-testing strategies
+      @minimum_update_period = Arke::Configuration.get(:min_update_period) || 1.0
```

Comment:
Maybe just call rate_limit
It's the API rate limit actually it should be a per exchange parameter